tests/reference-files/test_nurbs_hemisphere_surface_reference.dat

```diff
@@ -0,0 +1,558 @@
+// -----------------------------------------------------------------------------
+// This input file was created with MeshPy.
+// Copyright (c) 2018-2023
```

Comment:
This will cause an error with the most current master - should be `2018-2024` do you still have an old meshpy version?

Reply:
Most likely this is what caused the failing pipeline.

Reply:
I updated the meshpy version but forgot to update the test. Now it's solved.